utils/panel/settings/bool_setting.go

```diff
@@ -0,0 +1,135 @@
+package settings
```

Comment:
I feel like there is some repetition in the settings code. Is there any sensible way to get rid of it? Maybe having a `default` implementation and using a composition?